docs/fundamentals/code-analysis/style-rules/ide0078-ide0260.md

```diff
@@ -5,16 +5,18 @@ ms.date: 09/30/2020
 ms.topic: reference
 f1_keywords:
 - IDE0078
+- IDE0260
 - csharp_style_prefer_pattern_matching
 helpviewer_keywords:
 - IDE0078
+- IDE0260
 - csharp_style_prefer_pattern_matching
```

Comment:
Is `csharp_style_pattern_matching_over_as_with_null_check` missing in f1_keywords and helpviewer_keywords?